modules/friendlytag.js

```diff
@@ -1361,6 +1393,9 @@ Twinkle.tag.callback.evaluate = function friendlytagCallbackEvaluate(e) {
 				globalize: form["articleTags.globalize"] ? form["articleTags.globalize"].value : null,
 				notability: form["articleTags.notability"] ? form["articleTags.notability"].value : null
 			};
+			params.expertNeeded = form["articleTags.expertNeeded"] ? form["articleTags.expertNeeded"].value : null,
```

Comment:
What's the reason for putting these here — i.e., under `params` rather than `params.tag` (see L944-945)?  Should be removed from L1392.

Reply:
removed L1392 which was unused. Doesn't matter whether you put it into params.tagParameters or not, anyway in #487, I have fully gotten rid of this manual method of saving values in `params` object.

Reply:
Fair enough!